.github/workflows/ci.yml

```diff
@@ -9,6 +9,9 @@ on:
       - reopened
       - synchronize
       - ready_for_review
+  push:
+    branches:
+      - main
```

Comment:
Update the CHANGELOG comment about when the pipeline runs

Reply:
Updated